WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Service/BlogDashboardRemoteEntity.swift

```diff
@@ -75,10 +76,24 @@ extension BlogDashboardRemoteEntity {
             let icon: String?
         }
 
+        struct ImageSize: Decodable, Hashable {
+            let width: Int?
+            let height: Int?
+
+            var widthToHeightRatio: CGFloat {
+                CGFloat(
+                    width ?? 2
+                ) / CGFloat(
+                    height ?? 1
+                )
+            }
```

Comment:
[Suggestion] I think if either `width` or `height`, is `nil`. I would expect the aspect ration to be `nil` as well. Because it doesn't make sense to calculate the aspect ratio out of `nil` dimensions!

Reply:
Makes sense!